src/versioningInfo.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2008-2017 Pioneer Developers. See AUTHORS.txt for details
+// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+#include "versioningInfo.h"
+#include "libs.h"
+#include "utils.h"
+// open asset importer
+#include <assimp/version.h>
+// freetype
+#include "ft2build.h"
+#include FT_FREETYPE_H
+
+#ifdef ENABLE_SERVER_AGENT
+#include <curl/curlver.h>
+#endif
+
+void OutputVersioningInfo()
+{
+	SDL_version ver;
+	SDL_GetVersion(&ver);
+	Output("\n--------------------\n");
+	Output("SDL Version %d.%d.%d\n", ver.major, ver.minor, ver.patch);
+	Output("SDL_image ver: %d.%d.%d\n", SDL_IMAGE_MAJOR_VERSION, SDL_IMAGE_MINOR_VERSION, SDL_IMAGE_PATCHLEVEL);
```

Comment:
There's also the IMG_Linked_Version() function, which might be preferable.
SDL_IMAGE_*_VERSION is the version we've compiled against, IMG_Linked_Version should be the version being used at runtime.

Reply:
Ah interesting on both of those. 
I think it might be good to keep the compiled version and display what you're actually using at runtime so I'll add that.
It might highlight mismatches and other issues.

Reply:
You could do the same for SDL itself:

    Output("SDL Version (build) %d.%d.%d\n", SDL_MAJOR_VERSION, SDL_MINOR_VERSION, SDL_PATCHLEVEL);
    Output("SDL Version (dynamic) %d.%d.%d\n", ver.major, ver.minor, ver.patch);

Reply:
I was so tired the other day that I was actually trying to do this and didn't understand that I was already getting the dynamic library version 🙄 so tired and so glad that it's the holidays now!